docs/src/pages/components/snackbars/snackbars.md

```diff
@@ -5,7 +5,7 @@ components: Snackbar, SnackbarContent
 
 # Snackbars
 
-<p class="description">Snackbars provide brief messages about app processes through a message - typically at the bottom of the screen</p>
+<p class="description">Snackbars provide brief messages about app processes through a message - typically at the bottom of the screen.</p>
```

Comment:
Option 1:

```suggestion
<p class="description">Snackbars provide brief messages about app processes through a notification - typically at the bottom of the screen.</p>
```
?

(The spec just says "Snackbars provide brief messages about app processes at the bottom of the screen.", so not sure if our docs copied the v1 wording, or if it was augmented at some point.)

Option 2:

```suggestion
<p class="description">Snackbars provide brief messages about app processes - typically at the bottom of the screen.</p>
```

Reply:
Hmmm not sure the changes here. Originally there's a missing period at the end. But I'd prefer the second one :)

Reply:
Me too. You can accept changes with the "Commit suggestion" or "Add suggestion to batch" button. The latter groups them in one commit, but it doesn't matter either way, as we squash commits at merge.

Reply:
Thanks Matt. 👍 Will follow this process next time. I squashed the commits anyway kind of a habit :)